packages/server/src/routes.ts

```diff
@@ -113,6 +114,10 @@ export const createRouter = (ctx: AppContext): express.Router => {
   // Update or create a DID doc
   router.post('/:did', async function (req, res) {
     const { did } = req.params
+    // TODO: Provide carve-out for big clients like Bluesky
```

Comment:
as jake pointed out elsewhere, we generally use `x-ratelimit-bypass: <secret token>` as a means of getting around rate limits - that token can just be set in a env/config var for now & there's an example of how we do it over in the atproto repo

Reply:
I think I'd prioritize the earlier comment on only ratelimiting successful ops tho 🤔

Reply:
Yeah that sounds good; I added that support on the backend, but client support still needs to be added